agents/ppo.py

```diff
@@ -42,48 +48,57 @@ def create_from_config(cls, config):
             critic_loss_weight=config.get('critic_loss_weight', 1.0)
         )
 
-    def forward(self, state: State) -> Tuple[State, Policy, Value, Value]:
-        next_state, policy = self._actor.forward(state)
-        curr_v_func = self._critic.forward(state)  # (batch_size)
-        q_estimate = self._critic.forward(next_state)  # (batch_size)
-        # TODO [Vladimir Baikalov]: make same inerface for every agent's forward function
-        return next_state, policy, curr_v_func, q_estimate
+    def forward(self, inputs: Dict[str, Any]):
+        ppo_input = inputs[self._prefix]
+        next_state, policy = self._actor.forward(**ppo_input)
```

Comment:
No, I guess you don't get the idea. Here, we should pass the whole dict in the forward function. The idea is that this dict will also contain required inputs for `self._actor`. If something is missing we should implement preprocessing in this function before `self._actor(inputs)` call and put new fields in `inputs` dict.

```suggestion
        next_state, policy = self._actor(inputs)
```